aRchive.py

```diff
@@ -143,13 +151,27 @@ def archive_package_versions(bioc_pack, archive_dir):
             log.debug("Bioc_pack version %s %s" % (bioc_pack, curr_version))
             # Create new directory with version number as "-version" extension
             bioc_pack_name = os.path.split(bioc_pack)[-1]
-            output_directory = os.path.join(archive_dir,
-                                            bioc_pack_name, curr_version)
-            if not os.path.exists(output_directory):
-                log.info("Made new version directory: %s" % output_directory)
+            output_directory = os.path.join(archive_dir)
+            out_tarfile = bioc_pack_name + "_" + curr_version + '.tar.gz'
+            dest_tar_file = os.path.join(output_directory, out_tarfile)
+
+            log.info("\noutput_directory: %s \n out_tarfile: %s \n dest_tar_file: %s \n" % (
+                output_directory, out_tarfile, dest_tar_file))
+
+            if not os.path.exists(dest_tar_file):
                 # SAVE THE CURRENT VERSION HERE
-                shutil.copytree(bioc_pack, output_directory)
+                # Tar the directory
+                log.info('adding contents of bioc_pack %s to tarfile %s' % (bioc_pack, out_tarfile))
+                make_tarfile(out_tarfile, bioc_pack)
+                dest = os.path.join(output_directory, out_tarfile)
+                log.info('Moving %s to %s' % (out_tarfile, dest))
+                shutil.move(out_tarfile, dest)
```

Comment:
Just to prevent the Bioc base folder and the archive from mixing. No concrete reason otherwise.